app/javascript/components/MainContainer/Filters/Filters.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './filters.css'
+import Dropdown from 'react-dropdown'
+import 'react-dropdown/style.css'
+
+class Filters extends Component {
+    constructor(props){
+        super(props);
+
+        this.state = {
+            rating_filter_selected: 'all',
+            cuisine_filter_selected: 'all',
+            ten_bis_filter_selected: 'all'
+        }
+
+        this.onChangeRating = this.onChangeRating.bind(this);
+        this.onChangeCuisine = this.onChangeCuisine.bind(this);
+        this.onChangeTenBis = this.onChangeTenBis.bind(this);
+        this.onChangeSearch = this.onChangeSearch.bind(this);
+    }
+
+    onChangeRating(event){
+        this.setState({rating_filter_selected: event.value})
+        this.props.onChangeFilter('rating_filter', event.value);
+    }
```

Comment:
If you write these functions as:
```js
onChangeRating = (event) => {
  ...
};
```

You'll be able to avoid writing the whole `this.foo = this.foo.bind(this);` in the constructor.

This isn't an existing Javascript language feature, but there's support for it coming from the build tools.

Also, I suggest avoiding the use of the name `event`, because there's a global `window.event` variable.